client/my-sites/hosting/hosting-activate.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import MainComponent from 'components/main';
+import HeaderCake from 'components/header-cake';
+import EligibilityWarnings from 'blocks/eligibility-warnings';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import { initiateThemeTransfer } from 'state/themes/actions';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+const HostingActivate = ( { initiateTransfer, siteId, siteSlug, translate } ) => {
+	const getBackUrl = () => {
```

Comment:
This could be just a constant rather than a function:

```suggestion
	const getBackUrl = `/hosting-config/${ siteSlug }`;
```